src/devices/Hcsr501/Hcsr501.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Gpio;
+
+namespace Iot.Device.Hcsr501
+{
+    public class Hcsr501ValueChangedEventArgs : EventArgs
+    {
+        public readonly PinValue PinValue;
```

Comment:
we usually do this with property, i.e.: `public PinValue PinValue { get; private set; }`